Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -449,7 +449,7 @@ the elements of `α`. -/
 -- The explicit typing is required in order for `simp` to work properly.
 @[simps! symm_apply_coe]
 def enum (r : α → α → Prop) [IsWellOrder α r] :
-    @RelIso { o // o < type r } α (Subrel (· < ·) { o | o < type r }) r :=
+    @RelIso { o // o < type r } α (Subrel (· < ·) (Iio (type r))) r :=
```

Comment:
Is this change needed in this PR? If not, I'd suggest leaving it out of this one and doing it elsewhere, for a cleaner diff.

Reply:
This isn't needed here, I'll put it in another PR.